contracts/proxy/src/contract.rs

```diff
@@ -171,6 +178,47 @@ pub fn execute_relay(
     }
 }
 
+/// Add relayer to the relayers set
+pub fn execute_add_relayer(
+    deps: DepsMut,
+    info: MessageInfo,
+    relayer_addr: Addr,
+) -> Result<Response, ContractError> {
+    // Authorize user or multisig
+    authorize_user_or_multisig(deps.as_ref(), &info.sender)?;
```

Comment:
Perhaps needs to be `authorize_user_or_guardians`?  
Regardless if guardian is multisig, the guardian role should be able to do this.

Reply:
> Perhaps needs to be `authorize_user_or_guardians`? Regardless if guardian is multisig, the guardian role should be able to do this.

It makes sense, both guardian and multisig should able to do this.